deps/Makefile

```diff
@@ -414,7 +414,7 @@ LLVM_FLAGS += --enable-libcpp
 endif # USE_LIBCPP
 ifeq ($(OS), WINNT)
 LLVM_FLAGS += --with-extra-ld-options="-Wl,--stack,8388608" LDFLAGS=""
-LLVM_CPPFLAGS += -D__USING_SJLJ_EXCEPTIONS__ -D__CRT__NO_INLINE
+LLVM_CPPFLAGS += -D__USING_SJLJ_EXCEPTIONS__ -D__CRT__NO_INLINE -DMINGW_HAS_SECURE_API=1
```

Comment:
@tkelman To upgrade cygwin packages, login over SSH as the Administrator user, then run `/cygdrive/c/cygwin/setup-*.exe -q -n -R C:\cygwin -l C:\cygwin\packages -s http://mirrors.mit.edu/cygwin -g`.  Be sure to stop the buildslaves with `buildslave_stop` first, and then start them up again with `buildslave_start`.  I'm doing this for the two windows bots right now.